jaxws-ri/boms/bom/pom.xml

```diff
@@ -36,18 +36,19 @@
         <istack.version>4.0.1</istack.version>
         <gmbal-api.version>4.0.3</gmbal-api.version>
         <ha-api.version>3.1.13</ha-api.version>
-        <jaxb.version>3.0.2</jaxb.version>
-        <jaxws-api.version>3.0.1</jaxws-api.version>
-        <jakarta.jws-api.version>3.0.0</jakarta.jws-api.version>
+        <jaxb.version>4.0.0-M2</jaxb.version>
+        <jaxws-api.version>4.0.0-RC2</jaxws-api.version>
         <management-api.version>3.2.3</management-api.version>
         <mimepull.version>1.9.15</mimepull.version>
-        <saaj-api.version>2.0.1</saaj-api.version>
-        <saaj-impl.version>2.0.1</saaj-impl.version>
-        <streambuffer.version>2.0.2</streambuffer.version>
-        <woodstox-core.version>6.2.6</woodstox-core.version>
+        <resolver.version>20050927</resolver.version>
```

Comment:
resolver has been already removed in master

Reply:
removed